communications-handbook.md

```diff
@@ -0,0 +1,155 @@
+
+All US-RSE members should be invited to Slack.  New US-RSE members should be asked to join via [the link](https://forms.gle/CRsH7sKAk3UvZJfB9) on the main page of us-rse.org instead of only being invited to the Slack workspace so that there's a record of them and their email address is added to the list.  
+
+Members can create (and archive) channels, can create custom emojis, and do other Slack things that are part of the default settings.  The guideline is to be permissive with what members can do in the workspace until there is a need to implement tighter permissions to 1) enforce the code of conduct, or 2) manage the logistics of a growing membership list.  
+
+
+## us-rse.org Domain
+
+The us-rse.org domain is the US-RSE Assocation's home online, both in terms of the website and our organizational email addresses and groups/listservs. 
+
+Purpose:
+
+* Disseminate official information about the association
+* Conduct the business of the association
+* Host the main organizational website 
+* Host information related to association events and activities
+* Organizational email (run through a GSuite account)
+
+### Website
+
+Hosted via a GitHub repo and site: https://github.com/USRSE/usrse.github.io; #website channel on Slack
+
+Guidelines, procedures, etc. for the managing and updating the website are determined by the website committee and documented in the GitHub repo. The current guidelines for incremental changes require the use of a PR and a review of the PR. The guidelines are documented here  https://github.com/USRSE/usrse.github.io/blob/master/.github/CONTRIBUTING.md.  
+
+Large changes to the website including the creation of a new site, platform, subdomain, website section, etc. or a major reorganization or change in theme, styling, function of existing content must be first 1) discussed and agreed to by the Website Committee, and then 2) brought to the Steering Committee, before being added to the domain to ensure they are aligned with the mission of the Association. This includes changes to existing sites that significantly alter the way they function or content they provide. New sites or major changes to functionality should not be deployed under the us-rse.org domain before being reviewed. Instead, they should be previewed under a personal account or other resource. Contact the website committee for technical guidance. 
+
+### Email
+
+Distribution lists/groups: 
+
+* All US-RSE members are added to the announce@us-rse.org email list.  This is the list to be used for all important/official communications to the community.  Members of the Steering Committee can send to the list, but communications should generally come from the Steering Committee generally instead of individual members.
```

Comment:
```suggestion
* All US-RSE members are added to the announce@us-rse.org email list.  This is the list to be used for all important/official communications to the community.  Members of the Steering Committee can send to the list, but communications should generally come from the overall Steering Committee instead of individual members.
```